src/simple_http.rs

```diff
@@ -1,6 +1,7 @@
 //! This module implements a minimal and non standard conforming HTTP 1.0
 //! round-tripper that works with the bitcoind RPC server. This can be used
 //! if minimal dependencies are a goal and synchronous communication is ok.
+//!
```

Comment:
?

Reply:
I did this so every file has the same comment layout. This is copied from rust-bitcoin where I also did it to make everything uniform - pretty anal I know.  I did these docs fixes in preparation for bring this repo into the rust-bitcoin org.